### ISSUE TITLE
GStreamer error while building the app

### ISSUE THREAD

**Issue by liutingdu** [evercam/evercam-play-ios#1]
Got this error when building the app in Xcode: 

```
got the error:
duplicate symbol _ff_log2_tab in:
/Users/liutingdu/Library/Developer/GStreamer/iPhone.sdk/GStreamer.framework/GStreamer(libavcodec_a_arm64_-log2_tab.o)
/Users/liutingdu/Library/Developer/GStreamer/iPhone.sdk/GStreamer.framework/GStreamer(libavformat_a_arm64_-log2_tab.o)
duplicate symbol _ff_log2_tab in:
/Users/liutingdu/Library/Developer/GStreamer/iPhone.sdk/GStreamer.framework/GStreamer(libavcodec_a_arm64_-log2_tab.o)
/Users/liutingdu/Library/Developer/GStreamer/iPhone.sdk/GStreamer.framework/GStreamer(libavutil_a_arm64_-log2_tab.o)
duplicate symbol _iconv_locale_charset in:
/Users/liutingdu/Library/Developer/GStreamer/iPhone.sdk/GStreamer.framework/GStreamer(libcharset_a_arm64_-localcharset.o)
/Users/liutingdu/Library/Developer/GStreamer/iPhone.sdk/GStreamer.framework/GStreamer(libiconv_a_arm64_-localcharset.o)
duplicate symbol _hash_pjw_bare in:
/Users/liutingdu/Library/Developer/GStreamer/iPhone.sdk/GStreamer.framework/GStreamer(libgnutls-openssl_a_arm64_-hash-pjw-bare.o)
/Users/liutingdu/Library/Developer/GStreamer/iPhone.sdk/GStreamer.framework/GStreamer(libtasn1_a_arm64_-hash-pjw-bare.o)
duplicate symbol _strverscmp in:
/Users/liutingdu/Library/Developer/GStreamer/iPhone.sdk/GStreamer.framework/GStreamer(libgnutls-openssl_a_arm64_-strverscmp.o)
/Users/liutingdu/Library/Developer/GStreamer/iPhone.sdk/GStreamer.framework/GStreamer(libtasn1_a_arm64_-strverscmp.o)
ld: 5 duplicate symbols for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ld: 5 duplicate symbols for architecture armv7
......
```

Reported as a bug to GStreamer project: https://bugzilla.gnome.org/show_bug.cgi?id=749423

**Comment by liutingdu** [comment]
They've updated the bug status to be fixed: https://bugzilla.gnome.org/show_bug.cgi?id=749423

But seems the 1.5.90 binaries hasn't been updated yet: http://gstreamer.freedesktop.org/data/pkg/ios/

**Comment by liutingdu** [comment]
It was fixed in version 1.5.90's binary: http://gstreamer.freedesktop.org/data/pkg/ios/1.5.90/

The build issue could also be resolved if "all_load" flag is removed from target "Other Linker Flags".

I just re-added the video view in c29b69911a0ed9ad33b41a3f10c2679ee0a15df2
